lib/players/human.rb

```diff
@@ -0,0 +1,7 @@
+class Player::Human < Player
+
+  def move(move)
+    move = gets.chomp.upcase
+    move
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  gets.chomp.upcase
end
```